samples/attachments-fileshare/AttachmentsFileShare_1/Endpoint/fileToStream.txt

```diff
@@ -0,0 +1,12 @@
+﻿Call me Ishmael. Some years ago—never mind how long precisely—having little
```

Comment:
Is this text under public domain order under some copyright that would need to be mentioned?

Reply:
Herman Melville died over 100 years ago so [Moby Dick is in the public domain](https://en.wikisource.org/wiki/Moby-Dick).